pkg/clusterd/disk.go

```diff
@@ -80,7 +80,7 @@ func PopulateDeviceInfo(d string, executor exec.Executor) (*sys.LocalDisk, error
 	}
 
 	diskType, ok := diskProps["TYPE"]
-	if !ok || (diskType != sys.SSDType && diskType != sys.CryptType && diskType != sys.DiskType && diskType != sys.PartType && diskType != sys.LinearType) {
+	if !ok || (diskType != sys.SSDType && diskType != sys.CryptType && diskType != sys.DiskType && diskType != sys.PartType && diskType != sys.LinearType && diskType != sys.LVMType) {
```

Comment:
If the LVM disk is allowed here, it will also impact the non-PVC scenario. Is there any reason not to support this for the non-PVC scenario? This feature has been requested by others as also seen in #2047. Could you update this PR to support for the non-PVC scenario as well? Thanks!

Reply:
Not at all.  I didn't handle non-PVC scenario since I considered these are the different features.

I'll update this PR. Please wait for a while.

Reply:
thanks for looking into it!

Reply:
I'm ok if we need to address the non-PV scenario with a separate PR, but if it's not much difference it would be great to support at the same time. Were you able to investigate yet? If not, let's just focus on finishing this for feature complete for v1.2 next week. Thanks!